wagtail/wagtailadmin/static_src/wagtailadmin/scss/components/_streamfield.scss

```diff
@@ -57,52 +72,60 @@ li.sequence-member {
     .sequence-member-inner {
         @include clearfix;
         position: relative;
-        padding: 1.5em 50px;
+        padding: 20px 30px 50px 20px;
 
         .sequence-member {
-            border: 0;
+            // Second level nested Streamfield
+            border: 1px solid $color-teal;
+            border-bottom: 1px transparent;
+
+            &:first-child {
+                border-top-right-radius: 3px;
+                border-top-left-radius: 3px;
+            }
+
+            &:last-child {
+                border-bottom: 1px solid $color-teal;
+                border-bottom-right-radius: 3px;
+                border-bottom-left-radius: 3px;
```

Comment:
Make a variable out of the 3px radius to make it clearer what is going on between here and `:first-child`.